how-to/qemu-riscv.rst

```diff
@@ -35,53 +45,56 @@ Using the pre-installed server image
        :matches: (riscv64\.img)
 
    .. on jammy, use the +unmatched image for QEMU; later releases should use
-      the unsuffixed images, hence the horrid regex above
+      the unsuffixed images, hence the horrid regex above.
 
 #. Unpack the image:
 
    .. code-block:: text
 
-       xz -dk ubuntu-24.04.2-preinstalled-server-riscv64.img.xz
+       xz -dk ubuntu-preinstalled-server-riscv64.img.xz
 
 
 #. Optionally, if you want a larger disk, you can expand the disk (the
    file-system will be automatically resized too):
 
    .. code-block:: text
 
-       qemu-img resize -f raw ubuntu-24.04-preinstalled-server-riscv64.img +5G
+       qemu-img resize -f raw ubuntu-preinstalled-server-riscv64.img +5G
 
 
-#. Next use u-boot-qemu to boot the virtual machine. A working example with all
+#. Next use U-Boot to boot the virtual machine. A working example with all
    the options is:
 
    .. code-block:: text
 
        qemu-system-riscv64 \
            -cpu rva23s64 \
-           -machine virt -nographic -m 2048 -smp 4 \
+           -machine virt -m 4G -smp cpus=2 -nographic \
```

Comment:
Why do you want 4 GB of memory and only 2 CPUs?

Reply:
Well, that's a copy-paste of the last configuration in the docs. So, which one prefer?

- 4 CPUs + 2 GB RAM ([qemu-riscv.rst#L62](https://github.com/canonical/ubuntu-boards-documentation/blame/a76cfbf11b14af5dc21e8ed26e2aa15f8fcd64c9/how-to/qemu-riscv.rst?plain=1#L62))
- 4 CPUs + 4 GB RAM ([qemu-riscv.rst#L110](https://github.com/canonical/ubuntu-boards-documentation/blame/a76cfbf11b14af5dc21e8ed26e2aa15f8fcd64c9/how-to/qemu-riscv.rst?plain=1#L110))
- 2 CPUs + 4 GB RAM ([qemu-riscv.rst#L156](https://github.com/canonical/ubuntu-boards-documentation/blame/a76cfbf11b14af5dc21e8ed26e2aa15f8fcd64c9/how-to/qemu-riscv.rst?plain=1#L156))